File: leetcode/is_int_palidrome.py
def is_palindrome(x):
    to_string = str(x)
    first = 0
    last = len(to_string) - 1
    while first <= last:
        if to_string[first] is to_string[last]:
            first += 1
            last -= 1
        else:
            return False
    return True


print(is_palindrome(1011))

File: leetcode/isIntPalidrome.js
function isPalindrome(x) {
  const toString = x.toString();
  let front = 0;
  let back = toString.length - 1;
  while (front <= back) {
    if (toString[front] === toString[back]) {
      front++;
      back--;
    } else {
      return false;
    }
  }
  return true;
}

console.log(isPalindrome(101));
